core/src/test/scala/io/delta/tables/DeltaTableBuilderSuite.scala

```diff
@@ -366,4 +370,103 @@ class DeltaTableBuilderSuite extends QueryTest with SharedSparkSession with Delt
         e.getMessage.contains("is not a valid name"))
     }
   }
+
+  test("delta table property case") {
+
+    val preservedCaseConfig = Map("delta.appendOnly" -> "true", "Foo" -> "Bar", "foo" -> "Bar")
+
+    val lowerCaseEnforcedConfig = Map("delta.appendOnly" -> "true", "foo" -> "Bar")
+
+    sealed trait DeltaTablePropertySetOperation {
+
+      def setTableProperty(tablePath: String): Unit
+
+      def expectedConfig: Map[String, String]
+
+      def description: String
+
```

Comment:
nit: remove empty line

Reply:
I am a little bit confused, is the line 381 ok but the line 387 to be removed?

Reply:
Yep. We usually don't add an empty line between the last line and `}`.